CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [12.17.3]
+### Fixed
+- Stop emitting Dapp Viewed events on Firefox ([#32583](https://github.com/MetaMask/metamask-extension/pull/32583))
+- Make phishing warning events anonymous ([#32635](https://github.com/MetaMask/metamask-extension/pull/32635))
+- Keep showing the approve row in the batch transaction confirmation ([#32658](https://github.com/MetaMask/metamask-extension/pull/32658))
+- Support permit2 approvals in batch simulation ([#32733](https://github.com/MetaMask/metamask-extension/pull/32733))
+- Add metrics to check rejection of smart account update by user ([#32737](https://github.com/MetaMask/metamask-extension/pull/32737))
+- Delete preference to persist user rejection of upgrade ([#32697](https://github.com/MetaMask/metamask-extension/pull/32697))
```

Comment:
Not entirely happy with my wording choice here, but here's an attempt to explain that we're allowing users to upgrade who rejected the upgrade in the past:
```suggestion
- Allow users to upgrade to a smart contract account after rejecting it once in the past ([#32697](https://github.com/MetaMask/metamask-extension/pull/32697))
```

The previous wording was a little confusing to me (it says we're "deleting a preference", but the preference itself isn't something user-facing, it's strictly internal)